quickstarts/node-js/node-js/config.yml

```diff
@@ -22,7 +22,7 @@ description: |
   Our tool also provides other essential productivity and efficiency-enhancing innovations. Service Maps acquaints developers with their system's architecture providing vital insights to identify issues quickly. Error Analytics granularly pinpoints offending lines of code, relieving developers of a potentially arduous task so that they can concentrate on resolving issues. Our solution offers historical data that could prove essential in process improvement in addition to real-time information.
 summary: |
   New Relic's Node.js monitoring quickstart provides essential tools to monitor Node.js including multiple high-value alerts and informative dashboards to help developers visualize essential metrics and act on potential issues quickly.
-icon: logo.svg
+icon: logo.png
```

Comment:
any specific reason to change from svg to png ?

Reply:
Yes, previous logo is broken.
<img width="335" alt="Screenshot 2023-08-29 at 11 01 13 AM" src="https://github.com/newrelic/newrelic-quickstarts/assets/106150577/87c4e570-ff43-4b1e-8a48-7eefa0ea3843">